File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/page_plus_image_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PagePlusImageVO:
    total_elements: int = None
    total_pages: int = None
    size: int = None
    content: List[PlusImageVO] = None
    number: int = None
    first: bool = None
    last: bool = None
    number_of_elements: int = None
    sort: SortObject = None
    pageable: PageableObject = None
    empty: bool = None

File: sdkwork-backend-sdk-typescript/src/types/page-plus-image-vo.ts
import type { PageableObject } from './pageable-object';
import type { PlusImageVO } from './plus-image-vo';
import type { SortObject } from './sort-object';

export interface PagePlusImageVO {
  totalPages?: number;
  totalElements?: number;
  size?: number;
  content?: PlusImageVO[];
  number?: number;
  numberOfElements?: number;
  first?: boolean;
  last?: boolean;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}
